### ISSUE TITLE
The plugin keeps state of being updated, and does not update to the latest version

### ISSUE THREAD

**Issue by afshinhaghighat** [elixir-lsp/vscode-elixir-ls#153]
Hi.
I have been using this plugin for a while. After an update, the plugin repeatedly tries to update to version 0.6.1 and it is not successful.
However, this is really annoying, and makes it hard to work on the project.

I tried deleting `.elixir_ls` from all of my projects, but still no chance.

I have attached a GIF of the issue
![20201113_105707906](https://user-images.githubusercontent.com/72789532/99125622-b8f94280-2604-11eb-8c9a-5081e861fd03.gif)

Any ideas?

Thanks

**Comment by axelson** [comment]
Hmm, this would seem like it is a VSCode issue. Do you have problems installing or updating any other extensions? What happens if you uninstall and reinstall the extension? Have you checked the permissions of the folder that VSCode is installed into?

**Comment by afshinhaghighat** [comment]
The permissions seem fine. All other extensions are easily being updated without a flaw.
Uninstalling, reinstalling, disabling or any of the other actions does not do anything with it.
I uninstalled, removed `.elixir_ls` from all of the projects which had this folder in them, and reinstalled the extension. Still the same problem. It keeps staying on version 0.4.0, and trying to update to 0.6.1.
For now, I disabled Auto updates for extensions, at least not to have the loading status.

However there must be a solution for this, I think!
